isubit/luggage_isu
# LUGG-1012 Added patched flexslider

To test:
- Add an announcement
- Double-check to make sure play/pause button and bottom navigation circles are hidden
- Add another announcement
- Double-check to make sure play/pause button and bottom navigation circles are visible
